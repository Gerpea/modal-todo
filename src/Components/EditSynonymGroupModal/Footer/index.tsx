import React from 'react';

import Button from 'Containers/Button';

import style from '../styles.module.css';

export interface Props {
  children: React.ReactNode;
}

function Footer(): React.ReactElement {
  return (
    <div className={style.Footer}>
      <Button type="success">сохранить изменения</Button>
      <Button type="danger">очистить синонимы</Button>
    </div>
  );
}

export default Footer;
