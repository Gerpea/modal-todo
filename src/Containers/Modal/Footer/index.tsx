import React from 'react';

import style from '../styles.module.css';

export interface Props {
  children: React.ReactNode;
}

function ModalFooter({ children }: Props): React.ReactElement {
  return <div className={style.Footer}>{children}</div>;
}

export default ModalFooter;
