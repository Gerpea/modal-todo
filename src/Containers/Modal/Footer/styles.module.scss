@import '../../../variables';

.Footer {
  width: 100%;

  padding-top: 19px;
  border-top: 2px solid $c-gray-2;
}
