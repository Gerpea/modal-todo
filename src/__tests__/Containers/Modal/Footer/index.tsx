import React from 'react';
import renderer from 'react-test-renderer';

import Footer from 'Containers/Modal/Footer';

describe('ModalFooter test', () => {
  it('Footer renders', () => {
    const tree = renderer.create(<Footer>footer</Footer>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
