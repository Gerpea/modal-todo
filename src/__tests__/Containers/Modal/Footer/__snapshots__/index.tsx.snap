// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooter test Footer renders 1`] = `
<div
  className="Footer"
>
  footer
</div>
`;
